Formula/kibana.rb

```diff
@@ -13,21 +13,11 @@ class Kibana < Formula
     sha256 "21d01f2b1a4e462b0cf48443eb2334bea56f9ea8c4d5302202279924fa8499f4" => :high_sierra
   end
 
+  depends_on "python@3.8" => :build
   depends_on "yarn" => :build
-  depends_on :macos # Due to Python 2
   depends_on "node@10"
 
   def install
-    # remove non open source files
-    rm_rf "x-pack"
```

Comment:
Please keep this, it works now out of the box (the `inreplace`'s aren't needed anymore, because the build script now handles the case of missing `tsconfig.json`'s itself)!